tests/src/JIT/Performance/CodeQuality/SIMD/SeekUnroll/SeekUnroll.cs

```diff
@@ -143,6 +143,11 @@ public static bool Test(object boxedIndex)
     public static int Main()
     {
         int failures = 0;
+        if (!Vector.IsHardwareAccelerated)
+        {
```

Comment:
I added a comment and upped the value to 100000. We can up it again later if we need/want to. I've tested this inside runtest.cmd, and it runs successfully.